docs/creating_capsules/openvino_object_detector.rst

```diff
@@ -0,0 +1,219 @@
+       device_name=device
+   )
+
+Here we define a lambda function that creates an instance of a Backend class
+with the model and weights file, as well as the device this backend will run
+on. We will define this Backend class in the next section.
+
+.. code-block:: python
+
+   options = common_detector_options
+
+We give this capsule some basic options that are common among most all
+detector capsules.
+
+With this new ``capsule.py`` file added, your capsule directory should look
+like this:
+
+.. code-block::
+
+   detector_face_openvino
+   ├── capsule.py
+   ├── face-detection-adas-0001.bin
+   ├── face-detection-adas-0001.xml
+   └── meta.conf
+
+The Backend Class
+-----------------
+
+Finally, we will define the ``Backend`` class. This class defines how the
+capsule runs analysis on video frames. An instance of this class will be
+created for every device the capsule runs on. The ``Backend`` class doesn't
+have to be defined in any specific location, but we will add it to a new file
+called ``backend.py`` with the following contents:
+
+.. code-block:: python
+
+   from typing import Dict, List
+
+   import numpy as np
+
+   from vcap import (
+       DetectionNode,
+       Resize,
+       DETECTION_NODE_TYPE,
+       OPTION_TYPE,
+       BaseStreamState)
+   from vcap_utils import BaseOpenVINOBackend
+
+
+   class Backend(BaseOpenVINOBackend):
+       label_map: Dict[int, str] = {1: "face"}
+
+       def process_frame(self, frame: np.ndarray,
+                         detection_node: DETECTION_NODE_TYPE,
+                         options: Dict[str, OPTION_TYPE],
+                         state: BaseStreamState) -> DETECTION_NODE_TYPE:
+           input_dict, resize = self.prepare_inputs(frame)
+           prediction = self.send_to_batch(input_dict).get()
+           detections = self.parse_detection_results(
+               prediction, resize, self.label_map,
+               min_confidence=options["threshold"])
+           return detections
+
+Our ``Backend`` class subclasses from ``BaseOpenVINOBackend``. This backend
+handles loading the model into memory from the given files and provides utility
+methods that make writing OpenVINO backends easy. All we need to do is define
+the ``process_frame`` method. Let's take a look at each call in the method
+body.
+
+.. code-block:: python
+
+   input_dict, resize = self.prepare_inputs(frame)
+
+This line prepares the given video frame to be fed into the model. The video
+frame is resized to fit in the model and formatted in the way the model
+expects. Also provided is a resize object, which contains the necessary
+information to map the resulting detections to the coordinate system of the
+originally sized video frame.
+
+.. code-block:: python
+
+   prediction = self.send_to_batch(input_dict).get()
+
+Next, the input data is sent into the model for batch processing. We
+immediately start blocking until the result is ready with ``get``. The results
```

Comment:
Suggestion:
`We immediately start blocking` -> `The capsule will block and wait until`